src/Error/Error.php

```diff
@@ -207,7 +208,7 @@ public function getCategory()
         return $this->category;
     }
 
-    public function getSource() : ?Source
+    public function getSource(): ?Source
```

Comment:
I like this so much better.

Reply:
TBH I preferred the version before but Doctrine CS complies with PSR-12. I can live with that :)